pages/portfolio.html

```diff
@@ -1,12 +1,61 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="/stylesheets/styles.css" />
+    <link rel="stylesheet" href="/stylesheets/portfolio.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About Me</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="portfolio-container">
+      <h2>Ada Projects</h2>
+      <div class="grid-container">
+        <div>
+          <a href="https://github.com/maldonado-ivana/adagrams-py"
+            ><img src="/images/adagrams.png" alt="Adagrams project"
+          /></a>
+          <p>Adagrams</p>
+        </div>
+        <div>
+          <a href="https://github.com/maldonado-ivana/viewing-party"
+            ><img src="/images/viewing_party.png" alt="Viewing Party project"
+          /></a>
+          <p>Viewing Party</p>
+        </div>
+        <div>
+          <a href="https://github.com/maldonado-ivana/swap-meet"
+            ><img src="/images/swap-meet.png" alt="Swap meet project"
+          /></a>
+          <p>Swap Meet</p>
+        </div>
+        <div>
+          <a href="https://github.com/maldonado-ivana/task-list-api"
+            ><img src="/images/task_list.png" alt="Task List project"
+          /></a>
+          <p>Task List</p>
+        </div>
+      </div>
```

Comment:
Consider using an unordered list here instead of a div for a grid-container. 

Then you can have each project be a list item and then the a, p, image tag can be in the section (and they can be siblings too)

```HTML
<ul>
  <li>
    <section>
      <a href="https://github.com/maldonado-ivana/adagrams-py">
      <img src="/images/adagrams.png" alt="Adagrams project"/>
      <p>Adagrams</p>
    </section>
  </li>
  <li>
    <section>
       <a href="https://github.com/maldonado-ivana/viewing-party">
       <img src="/images/viewing_party.png" alt="Viewing Party project"/>
       <p>Viewing Party</p>
    </section>
  </li>
</ul>